src/components/HtmlFromRichText/index.js

```diff
@@ -48,22 +48,24 @@ const HtmlFromRichText = ({ content }) => {
           domNode.attribs.style = '';
         }
 
-        // Turn links into buttons
-        // waiting on
-        // if (
-        //   domNode.attribs.hasOwnProperty('href') &&
-        //   domNode.parent.name !== 'li'
-        // ) {
-        //   // replace the node with a button
-        //   return (
-        //     <a class="usa-button-primary" href={domNode.attribs.href}>
-        //       {/* this is kinda goofy, but the 'data' is the text of the link
-        //                and html-parser reads that as a child of 'a' */}
+        // If we're getting the button div from drafttail, we need to turn it into
```

Comment:
*lightbulb emoji*!

Reply:
have you made an issue for said 💡?

Reply:
@briaguya sorry, I meant that this 👇 is a good idea !